Stack.py

```diff
@@ -0,0 +1,45 @@
+class Stack():
+
+  def __init__(self):
+    pass
+
+  def is_stack_empty(self):
+    pass
+
+  def push(self, x):
+    pass
+
+  def pop(self):
+    pass 
+
+
+def test_stack():
+  #Create stack.
+  s = Stack()
+  
+  #Check if Stack is succesfuly created.
+  assert s != None, f'''The instance is not created successfully.'''
+
+  #Ckeck if s is empty.
+  assert s.is_stack_empty(), f'''At this point stack must be empty.'''
+
+  s_push_item_1 = s.push(x=4)
+  s_push_item_2 = s.push(x='letter') 
+
+  assert s.is_stack_empty() == False, f' Stack must not be empty at this point. '
+  
+  #Check if pop remove items correctly.
+  s_pop_element = s.pop() 
+  assert s_pop == s_push_element_2, f'Unespected output. '
+  s_pop_element = s.pop() 
+  assert s_pop == s_push_element_1, f'Unespected output. '
+
+  assert s.is_stack_empty() == True, f'At this point stack must be empty.'
+
+  try:
+    s.pop() 
+    assert s.is_stack_empty == True, f'Pop must not possible, stack must be empty.'
```

Comment:
1. by missing the parentheses at the end, the condition `s.is_stack_empty == True` is always verified.
2. The assert is useless, we checked it already at line 37.
3. Testing a condition with an assert inside a try-except is a really bad idea.

Please remove the assert line